2015-07-27 21:23:15,531 dbmanage.py [line:47] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('name', 'url', 'ip', 'whois', 'description') failed, reason is (1062, "Duplicate entry 'name' for key 'name'")
2015-07-27 21:30:34,566 dbmanage.py [line:47] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('name', 'url', 'ip', 'whois', 'description') failed, reason is (1062, "Duplicate entry 'name' for key 'name'")
2015-07-27 21:47:45,743 dbmanage.py [line:48] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('name', 'url', 'ip', 'whois', 'description') failed, reason is (1062, "Duplicate entry 'name' for key 'name'")
2015-07-27 21:50:11,128 dbmanage.py [line:48] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('name', 'url', 'ip', 'whois', 'description') failed, reason is (1062, "Duplicate entry 'name' for key 'name'")
2015-07-27 21:51:18,817 dbmanage.py [line:48] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('name', 'url', 'ip', 'whois', 'description') failed, reason is (1062, "Duplicate entry 'name' for key 'name'")
2015-07-27 22:10:44,088 dbmanage.py [line:48] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('a'a', 'aa', 'dfs', 'sdf', 'dfs') failed, reason is (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'a', 'aa', 'dfs', 'sdf', 'dfs')' at line 1")
2015-07-27 22:44:25,432 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('zyf', 'zyf', '3.3.3.3', '操', '轮奸') failed, reason is (1062, "Duplicate entry 'zyf' for key 'name'")
2015-07-27 22:45:53,543 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('zyf', 'zyf', '3.3.3.3', '操', '轮奸') failed, reason is (1062, "Duplicate entry 'zyf' for key 'name'")
2015-07-28 20:39:18,665 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('', '', '', '', '') failed, reason is (1062, "Duplicate entry '' for key 'name'")
2015-07-28 20:40:26,089 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('', '', '', '', '') failed, reason is (1062, "Duplicate entry '' for key 'name'")
2015-07-28 20:46:13,271 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('dfsdf', 'dsfsdf', 'sdfasdf', 'sdfa', 'sdf') failed, reason is (1062, "Duplicate entry 'dfsdf' for key 'name'")
2015-07-28 20:50:43,281 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('dfsdf', 'dsfsdf', 'sdfasdf', 'sdfa', 'sdf') failed, reason is (1062, "Duplicate entry 'dfsdf' for key 'name'")
2015-07-28 20:54:53,511 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('dfsdf', 'dsfsdf', 'sdfasdf', 'sdfa', 'sdf') failed, reason is (1062, "Duplicate entry 'dfsdf' for key 'name'")
2015-07-28 20:55:17,969 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('dfsdf', 'dsfsdf', 'sdfasdf', 'sdfa', 'sdf') failed, reason is (1062, "Duplicate entry 'dfsdf' for key 'name'")
2015-07-28 20:56:25,536 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('', '', '', '', '') failed, reason is (1062, "Duplicate entry '' for key 'name'")
2015-07-28 20:59:38,977 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('', '', '', '', '') failed, reason is (1062, "Duplicate entry '' for key 'name'")
2015-07-28 21:03:08,501 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('', '', '', '', '') failed, reason is (1062, "Duplicate entry '' for key 'name'")
2015-07-28 21:03:41,335 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('', '', '', '', '') failed, reason is (1062, "Duplicate entry '' for key 'name'")
2015-07-28 21:05:28,759 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('', '', '', '', '') failed, reason is (1062, "Duplicate entry '' for key 'name'")
2015-07-28 21:08:29,658 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('', '', '', '', '') failed, reason is (1062, "Duplicate entry '' for key 'name'")
2015-07-28 21:52:22,827 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('ss', 'ss', 'df', 'dd', 'ggg') failed, reason is (1062, "Duplicate entry 'ss' for key 'name'")
2015-07-28 21:53:21,541 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('ss', 'ss', 'df', 'dd', 'ggg') failed, reason is (1062, "Duplicate entry 'ss' for key 'name'")
2015-07-28 21:53:25,806 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('ss', 'ss', 'df', 'dd', 'ggg') failed, reason is (1062, "Duplicate entry 'ss' for key 'name'")
2015-07-28 21:53:27,134 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('ss', 'ss', 'df', 'dd', 'ggg') failed, reason is (1062, "Duplicate entry 'ss' for key 'name'")
2015-07-28 22:00:40,362 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('ss9', 'ss', 'df', 'dd', 'ggg') failed, reason is (1062, "Duplicate entry 'ss9' for key 'name'")
2015-07-28 22:03:09,529 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('ss9', 'ss', 'df', 'dd', 'ggg') failed, reason is (1062, "Duplicate entry 'ss9' for key 'name'")
2015-07-28 22:03:43,590 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('ss10', 'ss', 'df', 'dd', 'ggg') failed, reason is (1062, "Duplicate entry 'ss10' for key 'name'")
2015-07-28 22:04:04,716 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('ss11', 'ss', 'df', 'dd', 'ggg') failed, reason is (1062, "Duplicate entry 'ss11' for key 'name'")
2015-07-28 22:29:53,667 server.py [line:65] DEBUG error
2015-07-28 22:29:59,333 server.py [line:65] DEBUG error
2015-07-28 22:30:05,720 server.py [line:65] DEBUG error
2015-07-28 22:31:23,835 server.py [line:65] DEBUG error
2015-07-28 22:31:28,481 server.py [line:65] DEBUG error
2015-07-28 22:34:53,132 server.py [line:65] DEBUG error
2015-07-28 22:36:20,337 server.py [line:65] DEBUG error
2015-07-28 22:38:11,104 server.py [line:65] DEBUG error
2015-07-28 22:42:14,495 server.py [line:65] DEBUG error
2015-07-28 22:43:12,053 server.py [line:65] DEBUG error
2015-07-28 22:43:22,553 server.py [line:65] DEBUG error
2015-07-28 22:43:35,170 server.py [line:65] DEBUG error
2015-07-28 22:43:52,768 server.py [line:65] DEBUG error
2015-07-28 22:44:17,750 server.py [line:65] DEBUG error
2015-07-28 22:49:29,516 server.py [line:63] DEBUG error
2015-07-28 22:49:36,937 server.py [line:63] DEBUG error
2015-07-28 22:54:38,696 dbmanage.py [line:50] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('shengdongxu', 'ss1', 'df1', 'dd1', 'ggg') failed, reason is (1062, "Duplicate entry 'shengdongxu' for key 'name'")
2015-07-28 22:54:59,202 dbmanage.py [line:50] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('shengdongxu', 'ss1', 'df1', 'dd1', 'ggg') failed, reason is (1062, "Duplicate entry 'shengdongxu' for key 'name'")
2015-07-28 22:56:37,103 dbmanage.py [line:50] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('shengdongxu', 'ss1', 'df1', 'dd1', 'ggg') failed, reason is (1062, "Duplicate entry 'shengdongxu' for key 'name'")
2015-07-28 22:57:53,411 dbmanage.py [line:50] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('shengdongxu', 'ss1', 'df1', 'dd1', 'ggg') failed, reason is (1062, "Duplicate entry 'shengdongxu' for key 'name'")
2015-07-28 23:05:33,859 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('shengdongxu', 'ss1', 'df1', 'dd1', 'ggg') failed, reason is (1062, "Duplicate entry 'shengdongxu' for key 'name'")
2015-07-28 23:06:25,287 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('shengdongxumb', 'ss1', 'df1', 'dd1', 'ggg') failed, reason is (1062, "Duplicate entry 'shengdongxumb' for key 'name'")
2015-07-28 23:06:36,172 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('shengdongxumb1', 'ss1', 'df1', 'dd1', 'ggg') failed, reason is (1062, "Duplicate entry 'shengdongxumb1' for key 'name'")
2015-07-28 23:07:11,739 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('shengdongxumb1', 'ss1', 'df1', 'dd1', 'ggg') failed, reason is (1062, "Duplicate entry 'shengdongxumb1' for key 'name'")
2015-07-28 23:08:52,332 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('shengdongxumb2', 'ss1', 'df1', 'dd1', 'ggg') failed, reason is (1062, "Duplicate entry 'shengdongxumb2' for key 'name'")
2015-07-28 23:09:37,613 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('shengdongxumb3', 'ss1', 'df1', 'dd1', 'ggg') failed, reason is (1062, "Duplicate entry 'shengdongxumb3' for key 'name'")
2015-07-28 23:09:57,721 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('shengdongxumb3', 'ss1', 'df1', 'dd1', 'ggg') failed, reason is (1062, "Duplicate entry 'shengdongxumb3' for key 'name'")
2015-07-28 23:11:14,383 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('shengdongxumb4', 'ss1', 'df1', 'dd1', 'ggg') failed, reason is (1062, "Duplicate entry 'shengdongxumb4' for key 'name'")
2015-07-28 23:11:58,148 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('shengdongxumb4', 'ss1', 'df1', 'dd1', 'ggg') failed, reason is (1062, "Duplicate entry 'shengdongxumb4' for key 'name'")
2015-07-28 23:12:43,343 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('shengdongxumb3', 'ss1', 'df1', 'dd1', 'ggg') failed, reason is (1062, "Duplicate entry 'shengdongxumb3' for key 'name'")
2015-07-28 23:13:44,923 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('shengdongxumb5', 'ss1', 'df1', 'dd1', 'ggg') failed, reason is (1062, "Duplicate entry 'shengdongxumb5' for key 'name'")
2015-07-28 23:14:45,302 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('shengdongxumb6', 'ss1', 'df1', 'dd1', 'ggg') failed, reason is (1062, "Duplicate entry 'shengdongxumb6' for key 'name'")
2015-07-28 23:15:20,723 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('shengdongxumb7', 'ss1', 'df1', 'dd1', 'ggg') failed, reason is (1062, "Duplicate entry 'shengdongxumb7' for key 'name'")
2015-07-28 23:17:31,494 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('shengdongxumb8', 'ss1', 'df1', 'dd1', 'ggg') failed, reason is (1062, "Duplicate entry 'shengdongxumb8' for key 'name'")
2015-07-28 23:21:20,492 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('dfjk', 'dfkjslkjfa', 'skdjf;alskf', 'dfkk', 'jk') failed, reason is (1062, "Duplicate entry 'dfjk' for key 'name'")
2015-07-28 23:22:17,806 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('dfjk2', 'dfkjslkjfa', 'skdjf;alskf', 'dfkk', 'jk') failed, reason is (1062, "Duplicate entry 'dfjk2' for key 'name'")
2015-07-29 07:46:51,676 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('dfjk545', 'dfkjslkjfa', 'skdjf;alskf', 'w'er玩儿', 'jk') failed, reason is (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'er\xe7\x8e\xa9\xe5\x84\xbf', 'jk')' at line 1")
2015-07-29 07:47:14,773 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('dfjk54533', 'dfkjslkjfa', 'skdjf;alskf', 'w'er玩儿', 'jk') failed, reason is (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'er\xe7\x8e\xa9\xe5\x84\xbf', 'jk')' at line 1")
2015-07-29 20:08:26,196 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('dfjk54533', 'dfkjslkjfa', 'skdjf;alskf', 'w'er玩儿', 'jk') failed, reason is (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'er\xe7\x8e\xa9\xe5\x84\xbf', 'jk')' at line 1")
2015-07-29 20:08:39,927 dbmanage.py [line:49] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('dfjk545332', 'dfkjslkjfa', 'skdjf;alskf', 'w'er玩儿', 'jkdf') failed, reason is (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'er\xe7\x8e\xa9\xe5\x84\xbf', 'jkdf')' at line 1")
2015-07-29 20:13:44,633 dbmanage.py [line:52] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('dfjk54jjj', 'dfkjslkjfa', 'skdjf;alskf', 'wer玩儿', 'jkdf') failed, reason is (1062, "Duplicate entry 'dfjk54jjj' for key 'name'")
2015-07-29 20:26:57,810 dbmanage.py [line:52] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('test1', 'dfkjslkjfa', 'skdjf;alskf', 'wer玩儿', 'jkdf') failed, reason is (1062, "Duplicate entry 'test1' for key 'name'")
2015-07-29 20:27:38,755 dbmanage.py [line:52] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('test1', 'dfkjslkjfa', 'skdjf;alskf', 'wer玩儿', 'jkdf') failed, reason is (1062, "Duplicate entry 'test1' for key 'name'")
2015-07-29 20:28:10,986 dbmanage.py [line:52] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('test1', 'dfkjslkjfa', 'bbb', 'wer玩儿', 'jkdf') failed, reason is (1062, "Duplicate entry 'test1' for key 'name'")
2015-07-29 20:30:38,591 dbmanage.py [line:52] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('test1', 'dfkjslkjfa', 'bbb', 'wer玩儿', 'jkdf') failed, reason is (1062, "Duplicate entry 'test1' for key 'name'")
2015-07-29 20:40:25,129 dbmanage.py [line:52] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('test1', 'dfkjslkjfa', 'bbb', 'wer玩儿', 'jkdf') failed, reason is (1062, "Duplicate entry 'test1' for key 'name'")
2015-07-29 20:40:53,770 dbmanage.py [line:52] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('test1', 'dfkjslkjfa', 'bbb', 'wer玩儿', 'jkdf') failed, reason is (1062, "Duplicate entry 'test1' for key 'name'")
2015-07-29 20:44:54,686 dbmanage.py [line:52] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('test1', 'dfkjslkjfa', 'bbb', 'wer玩儿', 'jkdf') failed, reason is (1062, "Duplicate entry 'test1' for key 'name'")
2015-07-29 22:10:10,626 dbmanage.py [line:52] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('test1', 'dfkjslkjfa', 'bbb', 'wer玩儿', 'jkdf') failed, reason is (1062, "Duplicate entry 'test1' for key 'name'")
2015-08-01 13:54:31,693 dbmanage.py [line:51] ERROR Execute sql cmmmand delete form project where id=3 failed, reason is (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'project where id=3' at line 1")
2015-08-01 13:54:51,236 dbmanage.py [line:51] ERROR Execute sql cmmmand delete form project where id=3 failed, reason is (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'project where id=3' at line 1")
2015-08-01 13:57:03,419 dbmanage.py [line:51] ERROR Execute sql cmmmand delete from project where id=4 failed, reason is (2013, 'Lost connection to MySQL server during query')
2015-08-01 13:58:46,685 dbmanage.py [line:61] ERROR Find failed, command is select id,name from project order by 1, reason is (2006, 'MySQL server has gone away').
2015-08-01 13:59:16,444 dbmanage.py [line:61] ERROR Find failed, command is select id,name from project order by 1, reason is (2013, 'Lost connection to MySQL server during query').
2015-08-01 13:59:54,855 dbmanage.py [line:61] ERROR Find failed, command is select id,name from project order by 1, reason is (2006, 'MySQL server has gone away').
2015-08-03 21:19:46,997 dbmanage.py [line:51] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('aaa', 'aaa.com', '1.1.1.1', 'bbb', 'aaa des') failed, reason is (1062, "Duplicate entry 'aaa' for key 'name'")
2015-08-03 21:29:08,679 dbmanage.py [line:51] ERROR Execute sql cmmmand insert into project(name, url, ip, whois, description) values('aaa', 'aaa.com', '1.1.1.1', 'bbb', 'aaa des') failed, reason is (1062, "Duplicate entry 'aaa' for key 'name'")
2015-08-04 21:57:22,915 dbmanage.py [line:51] ERROR Execute sql cmmmand update project set name=ddd,url=ddd.com,ip=2.2.2.2,whois=ddd,description=aaa des where id=16 failed, reason is (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '.2.2,whois=ddd,description=aaa des where id=16' at line 1")
2015-08-04 21:58:23,575 dbmanage.py [line:51] ERROR Execute sql cmmmand update project set name=ddd,url=ddd.com,ip=2.2.2.2,whois=ddd,description=aaa where id=16 failed, reason is (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '.2.2,whois=ddd,description=aaa where id=16' at line 1")
2015-08-04 22:01:22,904 dbmanage.py [line:51] ERROR Execute sql cmmmand update project set name='ddd',url='ddd.com',ip='2.2.2.2',whois='ddd',description='aaa des'' where id='16' failed, reason is (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '16'' at line 1")
2015-08-04 22:02:20,515 dbmanage.py [line:51] ERROR Execute sql cmmmand update project set name='ddd',url='ddd.com',ip='2.2.2.2',whois='ddd',description='aaa des'' where id=16 failed, reason is (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''aaa des'' where id=16' at line 1")
2015-08-05 22:57:28,250 dbmanage.py [line:61] ERROR Find failed, command is select * from project where id=[object Object], reason is (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '[object Object]' at line 1").
2015-08-13 21:59:31,886 dbmanage.py [line:61] ERROR Find failed, command is select id,url,ip,level from host where project_id = 1 order by [object Object], reason is (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '[object Object]' at line 1").
2015-08-13 22:01:32,783 dbmanage.py [line:61] ERROR Find failed, command is select id,url,ip,level from host where project_id = 1 order by [object Object], reason is (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '[object Object]' at line 1").
